ut/utils.cpp

```diff
@@ -111,15 +114,16 @@ bool doPrintValue<ColumnArray, void>(const ColumnRef & c, const size_t row, std:
 template <>
 bool doPrintValue<ColumnTuple, void>(const ColumnRef & c, const size_t row, std::ostream & ostr) {
     if (const auto & tupple_col = c->As<ColumnTuple>()) {
-        ostr << "(";
+        std::stringstream sstr;
```

Comment:
not sure if this is necessary. can we just use the single ostr?

Reply:
Use the single ostr will make some format mistakes,like the following example:
```cpp
    cout << setw(10) << "(message1" << '\n';
    cout << setw(10) << "(";
    cout << "message2" << '\n';
```
This result is:
![image](https://github.com/user-attachments/assets/c0ef15cd-f8c4-4f16-8172-224c31bf6cd5)

We have to output all the messages at once to make sure format correct.